app/Http/Transformers/EventTransformer.php

```diff
@@ -16,9 +16,17 @@ class EventTransformer extends TransformerAbstract
     public function transform(Event $event)
     {
         return [
-            'event_id' => (string) $event->id,
+            'event_id' => $event->id,
```

Comment:
Actually, I think laravel returns it as a string anyway. We might have just been making it explicit. Disregard. 

I think in general things that are like identifiers should be strings since they don't behave like numbers that you would do math on or anything.